components/Layout.tsx

```diff
@@ -11,7 +11,10 @@ export function Layout({ children }: LayoutProps) {
   return (
     <div className={layoutStyles.container}>
       <header className={layoutStyles.header}>
-        <a href="/"> Latest updates</a>
+        <a className={GridStyles.animateddiv} href="/">
+          {" "}
```

Comment:
I think you may need to edit your Prettier settings or something to avoid these extra spaces being added, or you possibly have a habit of leaving extra spaces in HTML/JSX (i.e. `<div> Hello</div>` rather than `<div>Hello</div>`)